iOS_SDK/OneSignalSDK/SwiftPM/Public/Headers/OneSignal.h

```diff
@@ -0,0 +1,535 @@
+/**
```

Comment:
If we nest this file under `OneSignal` in `Headers` then the same Objective-C import statement (`#import <OneSignal/OneSignal.h>`) works like we have for our frameworks today. 
`iOS_SDK/OneSignalSDK/SwiftPM/Public/Headers/OneSignal/OneSignal.h`

I tested moving this file locally and it seems to work for Objective-C, and without causing any issue with Swift.

Reply:
made the change!

Reply:
👍